testutils/build.gradle

```diff
@@ -54,8 +94,23 @@ dependencies {
     implementation "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
     implementation 'androidx.appcompat:appcompat:1.0.2'
-    testImplementation 'junit:junit:4.12'
+    implementation "junit:junit:$rootProject.ext.junitVersion"
+    implementation "com.opencsv:opencsv:4.1"
+    compileOnly "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
+    annotationProcessor "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
     testImplementation "org.robolectric:robolectric:$rootProject.ext.robolectricVersion"
     androidTestImplementation 'androidx.test:runner:1.2.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    repackage 'com.microsoft.azure.kusto:kusto-ingest:2.1.0'
+    implementation tasks.repackageJar.outputs.files
+}
+
+static boolean getBoolean(final String s) {
+    if (s == null || s == "") {
```

Comment:
Slightly groovier

```groovy
static boolean getBoolean(final String s) {
    if (s?.trim()) {
        return s.equalsIgnoreCase("true");
    }

    return false;
}
```

Reply:
You might even be able to inline this entirely like `s?.toBoolean()` but my Groovy is a little rusty